imports/pages/groups/profile/index.js

```diff
@@ -60,16 +103,44 @@ class TemplateWithoutData extends Component {
     this.props.dispatch(modal.hide());
   }
 
-  sendRequest = (e, callback) => {
+  onPhoneNumberChange = (value: string) => {
+    const phoneNumber = value.replace(/[^\d]+/g, "");
+    return this.setState({ phoneNumber });
+  }
+
+  validatePhoneNumber = (value: string): boolean => {
+    if (value.length < 10) {
```

Comment:
`  validatePhoneNumber = (value: string): boolean => {
    if (value.length < 10) return false;
    return true;
  }
`

Like this? Or more one-liney?